src/app/report-rollup/report-rollup.service.ts

```diff
@@ -521,11 +530,13 @@ export class ReportRollupService {
   }
 
 
-  updateTreasureHuntResults(opportunitySummaries: Array<OpportunitySummary>, assessmentId: number) {
+  updateTreasureHuntResults(thuntResults: TreasureHuntResults, opportunityCardsData: Array<OpportunityCardData>, opportunitiesPaybackDetails: OpportunitiesPaybackDetails, assessmentId: number) {
     let currentResults: Array<TreasureHuntResultsData> = this.allTreasureHuntResults.value;
     let resultToBeUpdated: TreasureHuntResultsData = currentResults.find(result => { return result.assessment.id == assessmentId });
-    let updatedResults: TreasureHuntResults = this.treasureHuntReportService.calculateTreasureHuntResultsFromSummaries(opportunitySummaries, resultToBeUpdated.assessment.treasureHunt.currentEnergyUsage);
-    resultToBeUpdated.treasureHuntResults = updatedResults;
+    // let updatedResults: TreasureHuntResults = this.treasureHuntReportService.calculateTreasureHuntResultsFromSummaries(opportunitySummaries, resultToBeUpdated.assessment.treasureHunt.currentEnergyUsage);
```

Comment:
Comments here, in opportunity-summary html (ln 1, 2, and last line), as well as opportunity-summary css below.